### DISCUSSION TITLE
enable usage of custom Exporters using environment variable OTEL_TRACES_EXPORTER

### DISCUSSION THREAD

**Discussion by shadow0wolf** [open-telemetry/opentelemetry-java#5994] [Q&A]
I have implemented a custom exporter , i intend configure otel instrumented application to use this exporter without explicitly implementing AutoConfigurationCustomizerProvider and make otel instrumentation use it through "OTEL_TRACES_EXPORTER" environment variable.

**Comment by jkwatson** [comment]
I believe if you implement the `io.opentelemetry.sdk.autoconfigure.spi.traces.ConfigurableSpanExporterProvider` SPI interface, and make sure your custom exporter is on the classpath, then you can use whatever is returned from the `getName()` method in your env var and it should work.
